test/db-helper.js

```diff
@@ -1,8 +1,21 @@
 require('../app/data')
 const sequelize = require('../app/config/db')
+const { getCountry } = require('../app/lookups')
 
 const truncate = async () => {
```

Comment:
Suggest changing method name to 'truncateDatabase' or 'cleanDatabase' ?? 'truncate' suggests it operates on a single table perhaps. Only a suggestion though

Reply:
Got this one from the original Defra example, but agree ``truncateDatabase`` is clearer.